internal/schema/1.6/terraform.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package schema
+
+import (
+	"github.com/hashicorp/hcl-lang/lang"
+	"github.com/hashicorp/hcl-lang/schema"
+	"github.com/zclconf/go-cty/cty"
+)
+
+func patchTerraformBlockSchema(bs *schema.BlockSchema) *schema.BlockSchema {
+	bs.Body.Blocks["cloud"].Body.Blocks["workspaces"].Body = &schema.BodySchema{
+		Attributes: map[string]*schema.AttributeSchema{
+			"name": {
+				Constraint: schema.LiteralType{Type: cty.String},
+				IsOptional: true,
+				Description: lang.Markdown("The name of a single Terraform Cloud workspace " +
+					"to be used with this configuration When configured only the specified workspace " +
+					"can be used. This option conflicts with `tags`."),
+			},
+			"project": {
+				Constraint:  schema.LiteralType{Type: cty.String},
+				IsOptional:  true,
+				Description: lang.PlainText("The name of a project that resulting workspace(s) will be created in."),
```

Comment:
```suggestion
				Description: lang.PlainText("The name of a Terraform Cloud project. Workspaces that need creating will be created within this project."),
```
Just to align it more with the documentation:
https://developer.hashicorp.com/terraform/cli/v1.6.x/cloud/settings#project
(btw. there are two typos there also which I didn't copy here)

Reply:
I used the [Terraform codebase as source](https://github.com/hashicorp/terraform/blob/ae6b36b247f8cb48211401bc6680215e4ad0996d/internal/cloud/backend.go#L1277) as we did for the other attributes.

Would you prefer to always use the documentation in the future?

Reply:
Hmm, I would hope these are one and the same 😭 but apparently not here. It may be another thing to point out to Core when we also raise the typos. It should be pretty much the same IMHO.